terraform-modules/k8s-master/variables.tf

```diff
@@ -28,8 +28,14 @@ variable subnetwork {
   type = string
 }
 
+variable enable_master_authorized_networks {
+  type = bool
+  default = true
+}
+
 variable authorized_network_cidrs {
   type = list(string)
+  default = []
```

Comment:
If an empty list here + a `true` above = a totally unreachable cluster, do we instead want to treat `null` / empty list for this arg as a signal for "disable `master_authorized_networks`?

Reply:
I think a totally unreachable public master endpoint is a potentially desirable default configuration, security-wise. It's not a strong opinion though and my use case is not affected by implementing it the way you suggest so happy to make the change.

Reply:
Made the suggested change. Just to be clear, this means that now the default behavior if no authorized_network_cidrs are provided is to disable master_authorized_networks, meaning no restrictions on public endpoint access.